<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter_10">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>1111</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="maximumSize">
       <size>
        <width>11313166</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="maximumSize">
       <size>
        <width>393</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit"/>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="textEdit"/>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>Search File</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_2"/>
      <widget class="QPushButton" name="pushButton_5">
       <property name="styleSheet">
        <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="textEdit_2"/>
      <widget class="QSplitter" name="splitter_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>Search Scripts</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_Score_S"/>
   </widget>
   <widget class="QMenu" name="menuPersonl_P">
    <property name="title">
     <string>Personl(&amp;P)</string>
    </property>
    <addaction name="actionLcdColor_L"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuPersonl_P"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_O">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionLcdColor_L">
   <property name="text">
    <string>LcdColor(&amp;L)</string>
   </property>
  </action>
  <action name="actionSave_Score_S">
   <property name="text">
    <string>Save Score(&amp;S)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
